### ISSUE TITLE
Error client.set_url using firebase 

### ISSUE THREAD

**Issue by pythonfortinero** [Sightengine/client-nodejs#1]
code:
```
app.get('/imgvalidator', (req, res) =>{
  console.log(req.query.img);
  console.log(sightengine)
  sightengine.check(['nudity','wad','properties','celebrities','face-attributes']).set_url(req.query.img).then(function(result) {
    console.log("resultados", result);
    return res.send(result);
  }).catch(function(err) {
    console.log("error",err);
    return res.send(err);
  });
});

```

output:
error TypeError: error.json is not a function at fetch.then.catch (/user_code/node_modules/sightengine/sightengine.js:26:20) at process._tickDomainCallback (internal/process/next_tick.js:135:7)

**Comment by ghost** [comment]
Thank you for the feedback.
We have uploaded a new version (1.1.1), it will probably solve your problem.

Let me know :)

**Comment by pythonfortinero** [comment]
The problem was solved in the 1.1.1 version
